### PR TITLE
1534 combobox dropdown tests fixes

### PR DESCRIPTION
**Pull request by NataliaTikhomirova**

### CODE REVIEW

**Comment by mf-code** [comment]
[4462] - https://transfer.sh/8CEuX/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4462] - https://transfer.sh/9EIPT/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4462] - https://transfer.sh/olceR/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4462] - Allure report on Netlify: https://5e1db616ab33965f6a7b86e9--musing-swartz-6fc060.netlify.com

File: jdi-light-html-tests/src/test/java/io/github/epam/example/complex/MultiDropdownExampleTests.java [hunk 1]
```diff
@@ -34,6 +34,7 @@ public void selectTest() {
 
     @Test
     public void disabledTest() {
+        multiDropdown.check("Steam");
```
Comment (dexguitar) [inline jdi-light-html-tests/src/test/java/io/github/epam/example/complex/MultiDropdownExampleTests.java]: This line is obsolete, remove it.

**Pull request merged**